lib/activerecord/all/activerecord_test.rb

```diff
@@ -61,14 +61,13 @@ def change
         t.timestamps
       end
     end
- 
+
     say "Created a table"
- 
+
     suppress_messages { add_index :products, :name }
     say "and an index!", true
- 
+
     say_with_time 'Waiting for a while' do
-      sleep 10
```

Comment:
This was raising a type error when run in isolation:

```
lib/activerecord/all/activerecord_test.rb:71: Method sleep does not exist on ActiveRecordMigrationsTest https://srb.help/7003
    71 |      sleep 10
              ^^^^^^^^
    https://github.com/sorbet/sorbet/tree/80e1b24dadafc4ead575cb5e3166a691d2eb73e7/rbi/core/kernel.rbi#L800: Did you mean: Kernel#sleep?
     800 |  def sleep(duration); end
            ^^^^^^^^^^^^^^^^^^^
Errors: 1
```

I suspect that this is because we are no longer running with other gems at the same time. I viewed this as a positive result because the same issue could occur in RBI files and inadvertently cause a type error.

Reply:
@panthomakos - Would "Kernel.sleep 10" work?

Reply:
Yes, but the call is not run and does not impact the return value of the function so it's not necessary in the first place.